### DISCUSSION TITLE
BTCPay Server as payment platform

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#2176] [General]
Happy New Year everyone. In 2021, one of the goals is to better communicate BTCPay Server and its full potential. While in the early development stage it was the easiest to communicate BTCPay Server as "Bitcoin Payment Processor" the features we developed over the years and the ones that are coming make the software way more than that.

In one of the meetings  (animation call) @Kukks and brainstormed a bit on how we can better "market" or communicate BTCPay to people and reveal its full potential. 

BTCPay Server as a `modular BItcoin payment platform` was one of the concepts we came up with. 

BTCPay indeed becomes the platform that can be used in plenty of ways. I'm opening up this discussion to publicly share the concept and to hear if you guys have any other ideas. If the idea gets positive feedback, there are several first steps we can do:
- Change description of the btcpayserver.org
- Better communicate possibilities through documentation
- Change descriptions in BTCPay Server itself

**Comment by bolatovumar** [comment]
Sounds like a good idea to me in abstract. The only thing that worries me here is that it may be too early at this stage to call BTCPay a truly `modular Bitcoin payment platform`. I know we have plugins now and there is Transmuter as well but I don't believe the plugin ecosystem is that rich at the moment (correct me if I'm wrong). I feel like if the messaging were to change now it would create an expectation of a rich plugin ecosystem (like Wordpress for example) which is not quite there yet.

**Reply by pavlenex** [reply]
Good point. I should've been more clear, I was talking about the long-term goal of at least 12-months to make this change happen. I agree that at this moment `modular` may not be the most suitable adjective, but if all goes well, it should fit in nicely this year imo.

**Reply by Zaxounette** [reply]
We're all on the same page on what BTCPay can become in the short term.
As @bolatovumar said, we're not there yet.
Nonetheless, ACK the semantics change.

I also like the wording. `Modular Bitcoin payment platform` is clear, concise and should be understandable whatever user-type you are.

**Reply by pavlenex** [reply]
@Zaxounette Does the animated video which you guys are working on take this transition into consideration?

**Reply by Zaxounette** [reply]
> @Zaxounette Does the animated video which you guys are working on take this transition into consideration?

We have been iterating on ideas around the use-cases that are already available, not so much on the modularity yet.

I can add this as a point to be discussed explicitly starting next call as to **not** include a wording that limits BTCPay to the actual `payment processor` semantic.

**Reply by pavlenex** [reply]
Yes, that would be good. Focusing on more general usage rather than specifics gives us more ever-green video.

**Reply by Zaxounette** [reply]
Added this point for the next animation call (N°3) [here](https://github.com/btcpayserver/organization/issues/12#issuecomment-755382172/)

**Reply by dennisreimann** [reply]
My 2 sats: Removing the "Modular" takes away a bit of clunkiness and makes it sound better. "Bitcoin Payment Platform" sounds good.

**Reply by pavlenex** [reply]
Good point, we can probably mention  `Modularity, Extensibility` in the intro text, or if we find a simpler word that that that's easy to translate. But we should probably worry about that once we actually have that implemented, for now we don't. `Bitcoin Payment Platform` is easy to translate and understand.

**Reply by dennisreimann** [reply]
Agree, that feels more like additional info we could provide in a tagline :)

**Comment by Kukks** [comment]
Imo removing modular takes away the new vision. `Modular Bitcoin platform`
is my recommendation
